<?xml version="1.0"?>
<launch>
  <!-- Parameters to pass to the ROS node -->
  <arg name="move_group" default="manipulator"/>
  <arg name="ee_link" default="iiwa_link_ee"/>
  <arg name="ros_rate" default="10.0"/>
  
  <!--  This loads the robot description  -->
  <include file="$(find iiwa_examples)/launch/moveit_planning_drawing.launch">
    <arg name="sim" value="true"/>
    <arg name="world" value="iiwa.world"/>
  </include>
    
  <!-- Here we call the ROS node we have written, with the parameters on top   -->
  <node ns="iiwa" name="UpdownMoveit" pkg="iiwa_examples" type="updown_moveit" respawn="false" output="screen">
    <param name="move_group" value="$(arg move_group)"/>
    <param name="ee_link" value="$(arg ee_link)"/>
    <param name="ros_rate" value="$(arg ros_rate)"/>
  </node>
  
</launch>
